### PR TITLE
Fixed assign surface mesh manual flag when surface deviation is disabled

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/modules/Mesh.py [hunk 1]
```diff
@@ -350,9 +350,9 @@ def assign_surface_mesh_manual(self, names, surf_dev=None, normal_dev=None, aspe
         ----------
         names : list
             List of faces to apply the surface mesh to.
```
Comment (jeremich41) [inline pyaedt/modules/Mesh.py:352]: If we put a list of object it is working too.

Comment (jeremich41) [reply pyaedt/modules/Mesh.py:352]: So the doc can be "List of faces or directly objects to apply the surface mesh to."?

**Review by MaxJPRey — APPROVED** [review]

File: pyaedt/modules/Mesh.py [hunk 2]
```diff
@@ -382,9 +382,10 @@ def assign_surface_mesh_manual(self, names, surf_dev=None, normal_dev=None, aspe
         aspect_ratio_enable = 2
 
         if not surf_dev:
+            surf_dev_enable = 0
```
Comment (MaxJPRey) [inline pyaedt/modules/Mesh.py:385]: @maxcapodi78 I added some unit tests for the default value while "playing" with this feature and others.

Comment (maxcapodi78) [reply pyaedt/modules/Mesh.py:385]: great thanks

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**